environment_setup/configure_ai_vs_ai.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```

Comment:
maybe just worth calling this script `disable_ufw.sh` so its clear what the user is getting themselves into

Reply:
Agreed. Addressed.